ui/js/models/ph_series.js

```diff
@@ -0,0 +1,230 @@
+"use strict";
```

Comment:
Oh one other comment: factoring out a file like this is probably a good idea, but you should do that in a distinct commit. Moving stuff to a file while making changes at the same time makes it difficult to see exactly what changed when reviewing. I'd put this back into perf.js for now. We can move the code in a followup commit.